sample/src/main/java/org/aerogear/graphqlandroid/Utils.kt

```diff
@@ -26,8 +25,8 @@ import java.util.concurrent.Executor
 object Utils {
 
     //To run on emulator
-    const val BASE_URL = "http://192.168.0.101:4000/graphql"
-    private const val SQL_CACHE_NAME = "tasks3Db"
+    const val BASE_URL = "http://192.168.0.105:4000/graphql"
+    private const val SQL_CACHE_NAME = "tasks4Db"
```

Comment:
Why this is changed?

Reply:
Was trying something with the apollo cache but was not successful. While doing that changed to see some changes in the new cache generated.